dataflux_pytorch/benchmark/checkpointing/multinode/train.py

```diff
@@ -81,13 +104,21 @@ def main(project: str,
         model = DemoTransformer(vocab_size=dataset.vocab_size,
                                 nlayers=int(os.environ.get("NUM_LAYERS", 10)))
         new_path = os.path.join(ckpt_restore_path, f'ckpt_{i}.ckpt/')
-        dataflux_strategy = DatafluxFSDPStrategy(
-            path=new_path,
-            project_name=project,
-            storage_client=None,
-            model=model,
-            state_dict_type="sharded",
-        )
+        strategy = None
+        if args.strategy == DF_FSDP_STRATEGY:
+            print("Using DatafluxFSDPStrategy")
+            strategy = DatafluxFSDPStrategy(
+                path=new_path,
+                project_name=project,
+                storage_client=None,
+                model=model,
+                state_dict_type="sharded",
+            )
+        else:
```

Comment:
I would probably still evaluate the strategy here, and have a final else block raising an error if the strategy is invalid.

Reply:
Okay! I went with a simple if-else because the `strategy` flag only accepts two values and would error out if an invalid value is passed to it. But it's good to validate it here as well I think!

Reply:
You can set one of them as "default so it won't error out if omitted.

Reply:
Yes! I set the default value to the one that corresponds to `DatafluxFSDPStrategy`